jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SlidersTests.java

```diff
@@ -123,45 +143,94 @@ public void thumbSliderTest() {
     @Test(description = "Test checks slider's label")
     public void labelSliderTest() {
         disabledSlider.show();
-        disabledSliderControl.has().label();
+        disabledSlider.has().label("Disabled");
         thumbSlider.get(1).show();
         thumbSliderControl.get(1).has().noLabel();
+        inverseLabelSlider.show();
+        inverseLabelSlider.has().inverseLabel().and().label("Inverse label");
     }
 
     @Test(description = "Test checks slider's ticks")
     public void ticksSliderShowTicksWhenUsingSliderTest() {
+        String[] fruits = {"Figs", "Lemon", "Pear", "Apple"};
         Slider ticksSlider1 = ticksSlider.get(1);
-        Slider ticksSlider2 = ticksSlider.get(2);
         Slider ticksSlider4 = ticksSlider.get(4);
-        String[] fruits = {"Figs", "Lemon", "Pear", "Apple"};
+
+        colorsSlider.get(1).show();
+        colorsSlider.get(1).has().noTicks();
         ticksSlider1.show();
         ticksSlider1.core().actions((a, from) -> a.clickAndHold(from));
-        int totalTicks = ticksSlider1.ticks().size();
-        IntStream.range(0, totalTicks).forEach(index -> ticksSlider1.ticks().get(index + 1).is().displayed());
-        ticksSlider2.show();
-        ticksSlider2.is().tickAlwaysShow();
-        ticksSlider4.show();
-        IntStream.range(0, fruits.length).forEach(index -> ticksSlider.get(4).is().tickLabel(index + 1,
+        ticksSlider1.has().ticks();
+        ticksSlider1.ticks().forEach(tick -> tick.is().displayed());
+        ticksSlider.get(2).is().tickAlwaysShow();
+        ticksSlider.get(3).has().ticksSize(4);
+        IntStream.range(0, fruits.length).forEach(index -> ticksSlider4.is().tickLabel(index + 1,
                 fruits[index]));
     }
 
+    @Test(description = "Test checks slider's message: message(string)")
+    public void messageSliderTest() {
+        validationSlider.get(2).show();
+        validationSlider.get(2).has().messageText("40 in stock");
+    }
     @Test(description = "Test checks slider's error message", dataProvider = "validationSliderTestData", dataProviderClass = SliderTestsDataProvider.class)
     public void validationSliderTest(int index, double value, String expectedMessage) {
-        validationSlider.get(index).slideHorizontalTo(value);
+        validationSlider.get(index).setValue(value);
         validationSlider.get(index).has().value(value);
         waitCondition(() -> validationSlider.get(index).hasErrorMessages());
         validationSlider.get(index).has().errorMessage(expectedMessage);
     }
 
+    @Test(description = "Test checks slider's error messages: error (y/n)," +
+        "error-messages (string), rules")
+    public void errorSliderTest() {
+        List<String> errorMessages = Stream.of("2 is too low", "3 is too low")
+            .collect(Collectors.toList());
+        errorCountTextField.show();
```

Comment:
just in case :) possible solution for immutable lists List<String> list = List.of("1", "2");

Reply:
changed to ImmutableList.of